cloudinit/sources/DataSourceOVF.py

```diff
@@ -684,4 +740,73 @@ def _raise_error_status(prefix, error, event, config_file, conf):
     util.del_dir(os.path.dirname(config_file))
     raise error
 
+
+def load_cloudinit_data(metaPath, userPath):
+    """
+    Load the cloud-init meta data, user data, cfg and network from the
+    given files
+    """
+    LOG.debug('load meta data from: %s: user data from: %s',
+              metaPath, userPath)
+    md = {}
+    cfg = {}
+    ud = None
+    network = None
+
+    md = load(util.load_file(metaPath))
```

Comment:
I think we can drop both "def load and this callsite and just use util.read_conf(md_path) which will default to {} if file doesn't exist, is unparseable as YAML (emitting a more specific YAML format warnings in logs).

Reply:
great and I fixed it, PTAL.